core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -3370,45 +3369,43 @@ class EntityApiServiceSpec extends ApiServiceSpec {
       }
   }
 
-  // TODO CORE-635 Update quicksilver to have same behavior as legacy, then update test to use quicksilver
-  it should "return sorted results on entity query for number field" in withLegacyPaginationTestDataApiServices {
-    services =>
-      withStatsD {
-        Get(
-          s"${legacyPaginationTestData.workspace.path}/entityQuery/${legacyPaginationTestData.entityType}?sortField=number"
-        ) ~>
-          services.sealedInstrumentedRoutes ~>
-          check {
-            assertResult(StatusCodes.OK) {
-              status
-            }
-            assertResult(
-              EntityQueryResponse(
-                defaultQuery.copy(sortField = "number"),
-                EntityQueryResultMetadata(
-                  legacyPaginationTestData.numEntities,
-                  legacyPaginationTestData.numEntities,
-                  calculateNumPages(legacyPaginationTestData.numEntities, defaultQuery.pageSize)
-                ),
-                legacyPaginationTestData.entities
-                  .sortBy(_.attributes(AttributeName.withDefaultNS("number")).asInstanceOf[AttributeNumber].value)
-                  .take(defaultQuery.pageSize)
-              )
-            ) {
+  it should "return sorted results on entity query for number field" in withPaginationTestDataApiServices { services =>
```

Comment:
@calypsomatic this test was not failing for me after converting to Quicksilver. Did I do something wrong in the conversion?